src/test/java/com/cloudbees/jenkins/support/SupportPluginTest.java

```diff
@@ -103,4 +108,62 @@ public long getTime() throws IOException {
         assertNotNull(zip.getEntry("about.md"));
         assertNotNull(zip.getEntry("nodes.md"));
     }
+
+    /**
+     * Test that a component can supersede another component.
+     * This is similar to testGenerateBundleExceptionHandler() but we are superseding
+     * AboutJenkins and BuildQueue components. even if is added to the list of components to create.
+     * @throws Exception
+     */
+    @Test
+    public void testSupersedesComponent() throws Exception {
+        List<Component> components = Arrays.asList(
+                new Component() {
+                    @NonNull
+                    @Override
+                    public Set<Permission> getRequiredPermissions() {
+                        return Collections.singleton(Jenkins.ADMINISTER);
+                    }
+
+                    @NonNull
+                    @Override
+                    public String getDisplayName() {
+                        return "Test Component";
+                    }
+
+                    @Override
+                    public boolean supersedes(Component component) {
+                        return Set.of(AboutJenkins.class, BuildQueue.class).contains(component.getClass());
```

Comment:
perhaps more clearly just
```suggestion
                        return component instanceof AboutJenkins || component instanceof BuildQueue;
```